modules/_labs/select/react/lib/SelectMenu.tsx

```diff
@@ -136,13 +117,14 @@ const menuListBorderStyles = (theme: EmotionCanvasTheme, error?: ErrorType): CSS
 };
 
 const Menu = styled('div')<
-  Pick<SelectMenuProps, 'error' | 'isHiding' | 'shouldAnimate' | 'theme'> & {width: number}
+  Pick<SelectMenuProps, 'error' | 'theme' | 'visibility'> & {width: number}
 >(
   {
     backgroundColor: colors.frenchVanilla100,
     border: `1px solid ${inputColors.border}`,
     boxSizing: 'border-box',
     position: 'relative',
+    transition: `opacity ${menuAnimationDuration}ms`,
```

Comment:
I'm using CSS transitions now (instead of CSS animations) to better support interrupted animations (e.g., opening the menu again while it's closing).